k2/csrc/ragged_ops.cu

```diff
@@ -1249,14 +1250,62 @@ std::vector<RaggedShape> GetPrefixes(RaggedShape &src,
 
 Ragged<int32_t> AddSuffixToRagged(Ragged<int32_t> &src,
                                   const Array1<int32_t> &suffix) {
-  K2_LOG(FATAL) << "Not Implemented!";
-  return Ragged<int32_t>();
+  NVTX_RANGE(K2_FUNC);
+  K2_CHECK_EQ(suffix.Dim(), src.TotSize(src.NumAxes() - 2));
+  ContextPtr &c = src.Context();
+  Array1<int32_t> dst_values(c, src.NumElements() + suffix.Dim());
+  int32_t *dst_values_data = dst_values.Data();
+  int32_t num_axes = src.NumAxes();
+  const int32_t *src_row_splits_lastm1_data =
```

Comment:
I aslo suggest that you rename this `src/dst_row_split1_data` and `src/dst_row_ids1_data`, and add documentation to say that they are actually row_splits and row_ids on the last axis.
The advantage of naming `row_split1` and `row_ids1` is that we can use `idx01` and `idx0` to name those indexes in lambda, which will make code easy to read according to our naming convention